util/drakeGeometryUtil.cpp

```diff
@@ -564,14 +609,27 @@ Eigen::Matrix<typename Derived::Scalar, 3, 3> vectorToSkewSymmetric(const Eigen:
   return ret;
 }
 
+template <typename DerivedA, typename DerivedB>
+Eigen::Matrix<typename DerivedA::Scalar, 3, Eigen::Dynamic> dcrossProduct(
```

Comment:
should we update the dcross in matlab? (I also found it annoying when i went to use it recently).